### ISSUE TITLE
how to get test.txt

### ISSUE THREAD

**Issue by shaneshbi** [vsharecodes/percvae#1]
There is no "test_" related file in ConvAI2 dataset. So how to get test.txt ??
your Chapter 4 shows that the validation set is constructed from the training set. 
However there  exist  "validation set"  in ConvAI2.
Does that mean  you take the  "validation set"  in  ConvAI2  as test set in your experiment?

**Comment by haoyusoong** [comment]
Partition **train_self_original_no_cands.txt** for train.txt and valid.txt
Use **valid_self_original_no_cands.txt** for test.txt
More info in [./data](https://github.com/vsharecodes/percvae/tree/master/data)
And the test set in ConvAI2 is not publicly available, neither could we.
